ts/features/it-wallet/screens/issuing/pid/ItwPidAuthInfoScreen.tsx

```diff
@@ -70,78 +59,60 @@ const ItwPidAuthInfoScreen = () => {
     });
   };
 
+  /**
+   * Loading view component.
+   */
+  const LoadingView = () => (
+    <ItwLoadingSpinnerOverlay
+      captionTitle={I18n.t("features.itWallet.issuing.loading.title")}
+      captionSubtitle={I18n.t("features.itWallet.issuing.loading.subtitle")}
+      isLoading
```

Comment:
I think the title and subtitle are both wrong in this screen: 
![simulator_screenshot_6D3A8DA3-2CEC-49A1-A64D-49133F4A9167](https://github.com/pagopa/io-app/assets/12371664/e9510543-8ba1-4f98-bfcf-90f54475945a)

Reply:
I didn't find a match on the figma as there is no loading screen at this point. I used a title relevant to the type of credential which to date is only CIE. Could it be better to remove the title and subtitle? What do you think?

Reply:
Yes that's because we have to rethink the saga structure for the PID issuance flow given the new design. I think we can strip out the title and subtitle for now.